test-utils/build.gradle

```diff
@@ -24,17 +24,24 @@ dependencies {
         exclude group: 'org.slf4j', module: 'slf4j-reload4j'
     }
 
+
+    // The following dependencies needs to be excluded because they do not have licenses which we are allowed to include
+    def withoutLicenseViolations = {
+        exclude group: 'com.sun.codemodel'
+        exclude group: 'org.jline'
```

Comment:
This has a license: https://github.com/jline/jline3 , The license tool has issues sometimes if the repo owner hasn't declared it correctly, you can manually add it in to the licenses instead. See line 117 of licenses-3rdparties.gradle on how to do this.

Better to add in case something relies on these in Hadoop! :)